gaeguli/pipeline.c

```diff
@@ -1210,6 +1261,9 @@ gaeguli_pipeline_stop (GaeguliPipeline * self)
 
   g_debug ("clear internal pipeline");
 
+  /* Send SIGINT to worker pid */
+  kill (self->worker_pid, SIGINT);
```

Comment:
Maybe in the next change replace the signal with sending a terminate message through IPC? This could be a natural step towards actually implementing the IPC before the pipelines get moved into the subprocesses.